modules/tag/obs/suggest.js

```diff
@@ -0,0 +1,79 @@
+var nest = require('depnest')
+var {map, computed, watch} = require('mutant')
+const TagHelper = require('scuttle-tag')
+
+exports.needs = nest({
+  'about.obs.name': 'first',
+  'intl.sync.startsWith': 'first',
+  'keys.sync.id': 'first',
+  'sbot.obs.connection': 'first'
+})
+
+exports.gives = nest('tag.obs.suggest')
+
+exports.create = function (api) {
+  var suggestions = null
+  var tagsUsedByYou = null
+  var matches = api.intl.sync.startsWith
+
+  return nest('tag.obs.suggest', function (stagedTagIds) {
+    loadSuggestions(stagedTagIds)
+    return function (word) {
+      var filtered
+      if (!word) {
+        filtered = suggestions().slice(0, 200)
+      } else {
+        filtered = suggestions().filter((item) => {
```

Comment:
this method shouldn't be in tag.obs.suggest because it's not returning and obs, it's just returning a standard object as far as I can tell.

I would recommend making this method in `tag.obs.suggest`
I also think you should extract most of this out into ScuttleTag, and just have a skeleton left here so you can run loadSuggestions ad have it populating this `suggestions` cache. For myself I just make sure somewhere in the app I call this function to kick off the suggestions population.